### PR TITLE
WMCO-1156: Document WMCO proxy limitation

### PR DESCRIPTION
**Pull request by mburke5678**
https://github.com/openshift/windows-machine-config-operator/pull/1156/files

Previews: 
Added NOTE to
[Scheduling Windows container workloads](http://file.rdu.redhat.com/mburke/winc-cluster-proxy/windows_containers/scheduling-windows-workloads.html)
[Enabling Windows container workloads](http://file.rdu.redhat.com/mburke/winc-cluster-proxy/windows_containers/enabling-windows-container-workloads.html) 

Only these two WINC files are present as the topics are currently hidden in main awaiting the 6.0.0 release.

### CODE REVIEW

**Comment by mburke5678** [comment]
@sebsoto @jrvaldes In my limited understanding of the cluster-wide proxy, an [earlier draft of the statement by Sebastian](https://github.com/openshift/windows-machine-config-operator/pull/1156#pullrequestreview-1057458879) makes the most sense to me. PTAL

**Comment by mansikulkarni96** [comment]
> @mansikulkarni96 @sebsoto @jrvaldes In my limited understanding of the cluster-wide proxy, an [earlier draft of the statement by Mansi](https://github.com/openshift/windows-machine-config-operator/pull/1156#pullrequestreview-1057458879) makes the most sense to me. PTAL

@mburke5678 the earlier draft was not accurate as users are able to install the operator in a cluster with proxy setup. However, WMCO will not be able to route the traffic through the proxy connection for the workloads. Hence the final PR wording is more accurate.

**Comment by mburke5678** [comment]
@mansikulkarni96 I am not clear on what this means. 

> WMCO does not support adding Windows workloads using a cluster-wide proxy config for the OpenShift Container Platform.

Does it mean: WMCO is not supported in clusters that use a cluster-wide proxy. This is because WMCO is not able to route traffic through the proxy connection for the workloads.

**Comment by mansikulkarni96** [comment]
> @mansikulkarni96 I am not clear on what this means.
> 
> > WMCO does not support adding Windows workloads using a cluster-wide proxy config for the OpenShift Container Platform.
> 
> Does it mean: WMCO is not supported in clusters that use a cluster-wide proxy. This is because WMCO is not able to route traffic through the proxy connection for the workloads.

@mburke5678 yes that is a more concise statement. LGTM

**Comment by mburke5678** [comment]
@jfrancoa Is this an issue you can review? Thank you!

File: windows_containers/enabling-windows-container-workloads.adoc [hunk 1]
```diff
@@ -26,6 +26,11 @@ Dual NIC is not supported on WMCO-managed Windows instances.
 
 * You are running an {product-title} cluster version 4.6.8 or later.
 
+[NOTE]
+====
+The WMCO is not supported in clusters that use a xref:../networking/enable-cluster-wide-proxy.html[cluster-wide proxy] because the WMCO is not able to route traffic through the proxy connection for the workloads.
```
Comment (aireilly) [inline windows_containers/enabling-windows-container-workloads.adoc:31]: nit: ```suggestion
Windows Machine Config Operator (WMCO) is not supported in clusters that use a xref:../networking/enable-cluster-wide-proxy.html[cluster-wide proxy] because the WMCO is not able to route traffic through the proxy connection for the workloads.
```
If you agree with the change, apply in both notes.

Comment (aireilly) [inline windows_containers/enabling-windows-container-workloads.adoc:31]: I would suggest not using a definite article with Operator abbreviations. So, "WMCO is not supported...", not "the WMCO is not supported...". This probably comes down to preference. Search for "product name definite article" in the CCS gChat for further discussion.

Comment (mburke5678) [reply windows_containers/enabling-windows-container-workloads.adoc:31]: @aireilly Thanks. Do you have a link to that convo? I cannot seen to find it. (I am not much of a gChat user.)
If you don't mind, that sounds like a larger change, as there are many _The WMCO_ references in the docs.

Comment (bergerhoffer) [inline windows_containers/enabling-windows-container-workloads.adoc:31]: @mburke5678 Looks like the xref here uses the `.html` extension (should be `.adoc`) and is missing the #anchor. Can you submit another PR to fix this one up please?

FYI @aireilly heads up too

Comment (aireilly) [reply windows_containers/enabling-windows-container-workloads.adoc:31]: ack. Thanks @bergerhoffer
